config/optional/user.role.site_admin.yml

```diff
@@ -0,0 +1,214 @@
+langcode: en
```

Comment:
Why was this file created in the `optional` directory? This file duplicates the file with the same name in the `install` folder.